models/testdevice-1.0.x/go.mod

```diff
@@ -12,7 +11,5 @@ require (
 	github.com/openconfig/gnmi v0.0.0-20210914185457-51254b657b7d
 	github.com/openconfig/goyang v0.3.1
 	github.com/openconfig/ygot v0.12.4
```

Comment:
I think it would be good to change to `v0.14.0` here too and set `goyang` to `v0.4.0`
Also can do the same in `templates/go.mod.tpl`

Reply:
I tried setting just the ygot version and the unit tests didn't work. I'll try it with both versions set and see what happens